packages/ui/src/theme/tokens/components/stepperField.ts

```diff
@@ -54,7 +54,7 @@ export const stepperfield: StepperFieldTokens = {
     _disabled: {
       color: { value: '{components.button._disabled.color}' },
       backgroundColor: {
-        value: '{components.button._disabled.backgroundColor}',
+        value: '{components.fieldcontrol._disabled.backgroundColor}',
```

Comment:
Chose `components.fieldcontrol._disabled.backgroundColor` here such that when the entire stepper  component is disabled there is style consistency between the buttons and the field.